docker/docker-compose.yml

```diff
@@ -0,0 +1,120 @@
+version: '3.7'
+
+services:
+  mesatee-tms:
+    build:
+      context: ../
+      dockerfile: docker/Dockerfile.runtime.tms
+    ports:
+      - 5554:5554
+      - 5555
+    expose:
```

Comment:
Great work! 
I am just curious whether this docker-compose.yml can be auto generated from the config.toml, because it is a bit error-prone when modifying the ports and ip.

Reply:
I think this is okay because normally port numbers are always set as default. Other projects also have similar problem but I didn't find a better solution for now.